usaspending_api/api_contracts/contracts/agency/federal_obligations/FederalAccount.md

```diff
@@ -0,0 +1,74 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Agency Profile
```

Comment:
Could we update these headings & descriptions to be more specific to each new file? Something like 
```
Agency Profile - Minor Object Classes

This endpoint is used to power the object class visualization on USAspending.gov's agency profile pages. This data can be used to better understand the different ways that a specific agency spends money.
```

Reply:
That would be a better state for the code, but doing so in this PR would require me to interview  everyone for the context of each endpoint, delaying things for a while. I would advise moving that into further subtasks.

Reply:
Sure, I'd be happy to submit a follow-up PR since I'm familiar with how these are used.